config/clients/java/template/config-Configuration.java.mustache

```diff
@@ -63,13 +68,39 @@ public class Configuration implements BaseConfiguration {
         }
     }
 
+    /**
+     * Construct a new {@link Configuration} with any non-null values of a {@link ConfigurationOverride} and remaining values from this {@link Configuration}.
+     *
+     * @param configurationOverride The values to override
+     * @return A new {@link Configuration} with values of this Configuration mixed with non-null values of configurationOverride
+     */
+    public Configuration override(ConfigurationOverride configurationOverride) {
+        Configuration result = new Configuration(apiUrl);
+
+        String overrideApiUrl = configurationOverride.getApiUrl();
+        if (overrideApiUrl != null) {
```

Comment:
curious how come this is different than line 86/89 etc.

Reply:
I could update this to be symmetric and look the same. It might be more readable

Only for `apiUrl` I've already passed it in to a constructor (above, line 78) so there is no need to explicitly set it.

Reply:
I'll update it since I already introduced a no-args constructor.

Reply:
Done in [26f8b24](https://github.com/openfga/sdk-generator/pull/189/commits/26f8b248075be1430f69504975ba8b28d6519a1f)